frontend/packages/pipelines-plugin/src/components/repository/sections/RepositoryOverview.tsx

```diff
@@ -88,7 +98,27 @@ const RepositoryOverview = () => {
           {t('pipelines-plugin~Commit these changes and push them to your Git repository.')}
         </Text>
       </FormGroup>
-      <FormGroup fieldId="step-4">
+      {!(
+        values.gitProvider === GitProvider.GITHUB &&
+        values.method === GitProvider.GITHUB &&
+        values.githubAppAvailable === true
+      ) && (
+        <FormGroup fieldId="step-4">
+          <Title headingLevel="h4" size={TitleSizes.md}>
+            {t('pipelines-plugin~Step 4')}
+          </Title>
+          <Text>
+            <Trans t={t} ns="pipelines-plugin">
+              Webhook URL to configure the webhook in your{' '}
+              <a href={values.gitUrl}>Git repository</a> :
+            </Trans>
+            <ClipboardCopy variant={ClipboardCopyVariant.inlineCompact}>
+              {values?.webhook?.url}
+            </ClipboardCopy>
```

Comment:
Is values.webhook.url really optional? Then we should maybe add this to the condition for step 4?

And doesn't it make sense to show the webhook URL in a new line? As a non-inline element? Then the webhook URL would break like this:

![image](https://user-images.githubusercontent.com/139310/229838425-fb45ecd5-992d-4de7-90fb-317a24d0d3c5.png)